domains/schema/transforms/platformTransform.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from 'zod'
+
+const { formatMessage } = useIntl()
+
+/**
+ *
+ *
+ * @param values
+ * @param ctx
+ * @param type
+ */
+export const platformTransform = async (
+  values: {
+    src: string
+  },
+  ctx: z.RefinementCtx,
+  type: GridWidgetType
+) => {
+  try {
+    const parsedValues = await parsePlatformInput(type, values.src)
+    return parsedValues.properties
+  } catch (error: unknown) {
+    ctx.addIssue({
+      code: z.ZodIssueCode.custom,
+      message: formatMessage('errors_invalid_input', {
+        name: capitalize(type),
+      }),
+      path: ['src'],
+    })
```

Comment:
Why don't we use `error` object in the `addIssue`? It should have the information about which field is invalid.

Reply:
By `error` object you mean `ctx` instance? I've checked but it's empty in this case:

<img src="https://github.com/user-attachments/assets/df580cdd-a8c5-4f36-b2d5-7c928cde950d" width="400" />

Reply:
No, the `error: unknown`. 
It should return the information about the invalid field if such error is thrown.

Reply:
The error we catch here comes from `parsePlatformInput`, it doesn't have anything to do with zod unfortunatelly :/